lib/sales_engine.rb

```diff
@@ -11,6 +11,10 @@ def find_merchant_by_merchant_id(id)
     merchants.find_by_id(id)
   end
 
+  def find_items_by_id(id)
+    items.find_all_by_merchant_id(id)
+  end
+
   def analyst
     SalesAnalyst.new(self)
```

Comment:
Just a note, Megan mentioned that we do not have to have SalesAnalyst.new, and that we can just call .new here because it is a class method referring to self. We can make this minor refactor once we finish the iteration.

Reply:
Good Note! Let me know if this is a syntax convention we would like to follow and it will be a quick fix!